dev/io.openliberty.microprofile.telemetry.1.1.internal/src/io/openliberty/microprofile/telemetry11/internal/rest/TelemetryServletFilter.java

```diff
@@ -308,4 +309,9 @@ private boolean checkDisabled(Map<String, String> oTelConfigs) {
         return false;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public void destroy() {
+    }
+
```

Comment:
Any idea why this was needed?

Reply:
Newer versions of the interface have a default implementation of `destroy()` which does nothing.

We need to add this method to our implementation for compatibility with older versions.